### PR TITLE
ge(transaction-database): Accessing Databases in Camel Routes

### PR DESCRIPTION
**Pull request by jramcast**

### CODE REVIEW

File: transaction-database/solutions/payments/src/main/java/com/redhat/training/payments/PaymentAnalysisRouteBuilder.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.redhat.training.payments;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.stereotype.Component;
+
+// TODO: Enable the route by extending the RouteBuilder superclass
```
Comment (ebramirez) [inline transaction-database/solutions/payments/src/main/java/com/redhat/training/payments/PaymentAnalysisRouteBuilder.java:6]: Unnecessary in solutions.
```suggestion
```

File: transaction-database/solutions/payments/src/main/java/com/redhat/training/payments/PaymentAnalysisRouteBuilder.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+@Component
+public class PaymentAnalysisRouteBuilder extends RouteBuilder {
+
+    // TODO: Implement the configure method
```
Comment (ebramirez) [inline transaction-database/solutions/payments/src/main/java/com/redhat/training/payments/PaymentAnalysisRouteBuilder.java:10]: Unnecessary in solutions.
```suggestion
```

**Review by ebramirez — APPROVED** [review]
A couple of very minor suggestions.

File: transaction-database/apps/payments/src/main/java/com/redhat/training/payments/PaymentAnalysisRouteBuilder.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.redhat.training.payments;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.stereotype.Component;
+
+// TODO: Enable the route by extending the RouteBuilder superclass
```
[the remark this answers is not in the document]
Comment (jramcast) [reply transaction-database/apps/payments/src/main/java/com/redhat/training/payments/PaymentAnalysisRouteBuilder.java]: Right, I forgot to clean these comments. Good catch!

**Pull request merged**